parfive/downloader.py

```diff
@@ -445,20 +539,38 @@ def callback(token, future, main_pb):
                     else:
                         file_pb = None
 
-                    with open(str(filepath), 'wb') as fd:
-                        async for chunk in stream.iter_by_block():
-                            # Write this chunk to the output file.
-                            fd.write(chunk)
+                    queue = asyncio.Queue()
```

Comment:
Same comment about the name of this queue applies here as it does in `_get_http`.